### PR TITLE
release: update manifest and helm charts for v0.2.0

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/secrets-store-csi-driver/templates/_helpers.tpl [hunk 1]
```diff
@@ -12,13 +12,17 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "sscd.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
```
Comment (tam7t) [inline charts/secrets-store-csi-driver/templates/_helpers.tpl:17]: Is this the result of running `make promote-staging-manifest`?

Comment (aramase) [reply charts/secrets-store-csi-driver/templates/_helpers.tpl:17]: yeah, this change was added in https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/671

**Comment by aramase** [comment]
/hold

for #690 to merge

File: charts/secrets-store-csi-driver/templates/_helpers.tpl (hunk 1 above)
Comment (tam7t) [reply charts/secrets-store-csi-driver/templates/_helpers.tpl:17]: ah i was forgetting the difference between the charts vs staging folders again

**Pull request merged**